hibernate-core/src/main/java/org/hibernate/procedure/internal/ProcedureCallImpl.java

```diff
@@ -1085,7 +1091,7 @@ public Query<R> descending(int element) {
 	}
 
 	@Override
-	public Query<R> unordered() {
+	public Query<R> clearOrder() {
```

Comment:
Shouldn't this be `clearSort` if your other method is called `sort`?

Though TBH I find `clearOrder` + `orderBy(SingularAttribute, SortOrder)` would be closer to SQL/HQL and thus clearer/more consistent.

Reply:
I agree. I prefer "order", which is what I started with. But then I would need to rename:

- `Sort` -> `Order`
- `SortOrder` -> ????

I need a new name for `SortOrder`.

Reply:
At one stage I tried calling it just `Direction` but that was a bit too generic. `OrderDirection` seems like it fits but for some reason I can't quite put my finger on it sounds weird to me. "Clumsy" as you say.

Reply:
It's `SortDirection`.